changelog/fix-docblock-tribe_get_listview_args

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: tweak
+
+Add information and missing tags to �+� `tribe_get_listview_args()` docblock.
```

Comment:
`Â+Â` what are these chars ? Maybe they are there by mistake ?

Reply:
These got in there when I copy/pasted the chengelog line. I will make sure I check next time.